.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - "stable"
+cache:
+  directories:
+  - node_modules
```

Comment:
![remember this](https://user-images.githubusercontent.com/517302/45968221-441ab700-bffe-11e8-913a-d375448c066b.gif)